runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -173,7 +170,7 @@ public Query query(@Nonnull final String namespace) {
     void subscribe(@Nonnull StreamListener streamListener, @Nonnull String namespace,
                    @Nonnull List<TableSchema> tablesOfInterest,
                    @Nullable Timestamp timestamp) {
-        txnStreamingManager.subscribe(streamListener, namespace, tablesOfInterest,
+        streamManager.subscribe(streamListener, namespace, tablesOfInterest,
                 (timestamp == null) ? getTimestamp().getSequence() : timestamp.getSequence());
```

Comment:
is the client expected to know the start address of the stream of interest?  Is it possible for the client to have this info in a clean way?

Reply:
CorfuStore exposes an api called getTimestamp() which abstracts the internal start address of the stream. Please check out Timestamp in CorfuStoreMetadata.